src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/JJDisputedCount.java

```diff
@@ -81,37 +81,111 @@ public class JJDisputedCount extends Auditable<String> {
 	@Schema(nullable = false)
 	@Enumerated(EnumType.STRING)
 	private YesNo appearInCourt;
-	
+
+	@Column
+	private Long appearanceChargeCountId;
+
+	@Column
+	private Long courtAppearanceId;
```

Comment:
courtAppearanceId cannot be a required field as there might be any connection from an appearance charge count to a court appearance when it is first created.

Reply:
not sure how its noted in Java, maybe the way you have it is correct but nearly all fields in this table should be nullable.

Reply:
By default, fields annotated via @Columns are nullable, unless we specified not null explicitly, i.e.
@Column(nullable = false)